### DISCUSSION TITLE
[Feature] [Mobile] Ability to auto-upload certain device folders/albums

### DISCUSSION THREAD

**Discussion by nar3n12** [immich-app/immich#6315] [Feature Request]
### The feature

Hello Immich community,
I have been using Immich for a few days. I really like it. I used to run Photoprism previously, but it didn't suit for my needs. Immich perfectly matches my needs, very similar to the photo management app I'm used to. 

The feature:
User can enable auto-backup only for selected folders, or ignore selected folders from auto-backup. This way I can auto-upload my camera folder, while still be able to manually upload photos from other folders either one by one or a whole album/folder. This way it essentially becomes the single gallery app, if I have selected all the device albums to backup, but only enable auto-backup on Camera folder.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by nar3n12** [comment]
@bo0tzz hey, can you please give a link to the original issue to which this is duplicate of, so that I can follow. Thanks

**Reply by bo0tzz** [reply]
It is already possible to include/exclude specific albums for backup.

**Reply by nar3n12** [reply]
I think I didn't make it clear. I meant to say, select multiple albums(say 4) for backup, so they all will show up as device albums in immich, but auto-backup for only 1 album. 

It is already possible to select either auto-backup all selected device albums or none.
